modules/lvgl/lvgl_display.c

```diff
@@ -45,6 +45,24 @@ void lvgl_wait_cb(lv_disp_drv_t *disp_drv)
 
 #endif /* CONFIG_LV_Z_FLUSH_THREAD */
 
+#ifdef CONFIG_LV_Z_USE_ROUNDER_CB
+void lvgl_rounder_cb(lv_disp_drv_t *disp_drv, lv_area_t *area)
```

Comment:
You could add a define here in the opposite case to reduce the number of `ifdef`s:

```c
#ifdef CONFIG_LV_Z_USE_ROUNDER_CB
void lvgl_rounder_cb(lv_disp_drv_t *disp_drv, lv_area_t *area)
{
	// body
}
#else
#define lvgl_rounder_cb NULL
#endif

// No need to `ifdef` in the assignment anymore:
		disp_drv->rounder_cb = lvgl_rounder_cb;
```